nengo_dl/tests/test_converter.py

```diff
@@ -0,0 +1,460 @@
+# pylint: disable=missing-docstring
+
+import nengo
+import numpy as np
+import pytest
+import tensorflow as tf
+
+
+from nengo_dl import converter, utils
+
+
+def _test_convert(inputs, outputs, allow_fallback=False, inp_vals=None):
+    model = tf.keras.Model(inputs=inputs, outputs=outputs)
+
+    conv = converter.Converter(model, allow_fallback=allow_fallback)
+
+    conv.verify(training=False, inputs=inp_vals)
```

Comment:
If you make the `return True` change for `verify` that I suggested, you should also `assert conv.verify(...)` here and in the rest of this file, as it's a little weird that this file has very few (no?) asserts :+1: